tests/inventory/pipelines/test_data/fake_gke_services.py

```diff
@@ -153,32 +153,14 @@
      'monitoring_service': 'none',
      'name': 'redditmobile-canary-central',
      'network': 'default',
      'node_config': '{"diskSizeGb": 100, "machineType": "n1-standard-1", "oauthScopes": ["https://www.googleapis.com/auth/compute", "https://www.googleapis.com/auth/devstorage.read_only", "https://www.googleapis.com/auth/logging.write", "https://www.googleapis.com/auth/servicecontrol", "https://www.googleapis.com/auth/service.management.readonly", "https://www.googleapis.com/auth/trace.append"], "imageType": "COS", "serviceAccount": "default"}',
      'node_ipv4_cidr_size': 24,
-     'node_pools': [{'autoscaling': {},
-                     'config': {'diskSizeGb': 100,
-                                'imageType': 'COS',
-                                'machineType': 'n1-standard-1',
-                                'oauthScopes': ['https://www.googleapis.com/auth/compute',
-                                                'https://www.googleapis.com/auth/devstorage.read_only',
-                                                'https://www.googleapis.com/auth/logging.write',
-                                                'https://www.googleapis.com/auth/servicecontrol',
-                                                'https://www.googleapis.com/auth/service.management.readonly',
-                                                'https://www.googleapis.com/auth/trace.append'],
-                                'serviceAccount': 'default'},
-                     'initialNodeCount': 1,
-                     'instanceGroupUrls': ['https://www.googleapis.com/compute/v1/projects/cicd-prod/zones/us-central1-b/instanceGroupManagers/gke-redditmobile-canary--default-pool-1388e059-grp'],
-                     'management': {},
-                     'name': 'default-pool',
-                     'selfLink': 'https://container.googleapis.com/v1/projects/cicd-prod/zones/us-central1-b/clusters/redditmobile-canary-central/nodePools/default-pool',
-                     'status': 'RUNNING',
-                     'version': '1.4.5'}],
+     'node_pools': '[{"status": "RUNNING", "version": "1.4.5", "name": "default-pool", "instanceGroupUrls": ["https://www.googleapis.com/compute/v1/projects/cicd-prod/zones/us-central1-b/instanceGroupManagers/gke-redditmobile-canary--default-pool-1388e059-grp"], "management": {}, "autoscaling": {}, "config": {"diskSizeGb": 100, "machineType": "n1-standard-1", "oauthScopes": ["https://www.googleapis.com/auth/compute", "https://www.googleapis.com/auth/devstorage.read_only", "https://www.googleapis.com/auth/logging.write", "https://www.googleapis.com/auth/servicecontrol", "https://www.googleapis.com/auth/service.management.readonly", "https://www.googleapis.com/auth/trace.append"], "imageType": "COS", "serviceAccount": "default"}, "selfLink": "https://container.googleapis.com/v1/projects/cicd-prod/zones/us-central1-b/clusters/redditmobile-canary-central/nodePools/default-pool", "initialNodeCount": 1}]',
```

Comment:
The names of the project and gke cluster and node pools in the test should be better anonymized.